.gitattributes

```diff
@@ -4,3 +4,4 @@ apps/bare-expo/ios/BareExpo.xcodeproj/project.pbxproj merge=union -text diff lin
 packages/*/build/** -diff linguist-generated
 packages/*/plugin/build/** -diff linguist-generated
 packages/@*/*/build/** -diff linguist-generated
+*.webp filter=lfs diff=lfs merge=lfs -text
```

Comment:
Do we need this one? 🤔

Reply:
https://github.com/expo/expo/pull/13416#discussion_r660148526